x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/eql/trial_license_complete_tier/eql.ts

```diff
@@ -247,32 +246,23 @@ export default ({ getService }: FtrProviderContext) => {
       ).eql(1);
     });
 
-    // Failing: See https://github.com/elastic/kibana/issues/209024
-    it.skip('parses shard failures for EQL event query', async () => {
+    it('parses shard failures for EQL event query', async () => {
       await esArchiver.load(packetBeatPath);
       const rule: EqlRuleCreateProps = {
         ...getEqlRuleForAlertTesting(['auditbeat-*', 'packetbeat-*']),
         query: 'any where agent.type == "packetbeat" or broken == 1',
       };
       await setBrokenRuntimeField({ es, index: 'auditbeat-*' });
-      const createdRule = await createRule(supertest, log, rule);
-      const createdRuleId = createdRule.id;
-      await waitForRulePartialFailure({ supertest, log, id: createdRuleId });
-      const route = routeWithNamespace(DETECTION_ENGINE_RULES_URL);
-      const response = await supertest
-        .get(route)
-        .set('kbn-xsrf', 'true')
-        .set('elastic-api-version', '2023-10-31')
-        .query({ id: createdRule.id })
-        .expect(200);
-
-      const ruleResponse = response.body;
+      const { logs } = await previewRule({ supertest, rule });
       expect(
-        ruleResponse.execution_summary.last_execution.message.includes(
-          'The EQL event query was only executed on the available shards. The query failed to run successfully on the following shards:'
+        logs.some((previewLog) =>
```

Comment:
Nit: this test is just comparing `true` to a boolean, so the failure message is just going to be "expected 'false' to equal 'true'."

It's a little more verbose, but we can get a much better failure message by using `expect.arrayContaining` and `expect.objectContaining`:

```ts
expect(logs).toEqual(expect.arrayContaining([
  expect.objectContaining({
    warnings: expect.arrayContaining([expect.stringContaining('The EQL ...')])
  })
]);
```

Reply:
Here's the issue: https://github.com/elastic/kibana/issues/215165